### PR TITLE
[docs]: Fix documentation for SVG Image component

### LINKED ISSUE #11116: [docs]: Update the documentation for the SVG Image Component
### The entire URL of the documentation with the issue

https://docs.tooljet.com/docs/widgets/svg-image

**Content updates:**
- Layout Table
  - Update the title name from "Layout" to "Devices" [reference](https://docs.tooljet.com/docs/widgets/text-input/#devices)
  - Update the name of the first column from  "Layout" to "Property"
  - Update the content in the "Description" and "Expected value" columns as per the given [reference](https://docs.tooljet.com/docs/widgets/text-input/#devices) ( Please just check the sentence structure from the given reference )
  - Add a full stop at the end of the sentence in the "Description" column.
- Add a divider before the "Styles" section [reference](https://docs.tooljet.com/docs/widgets/textarea#styles)
- Style Table
   -  Update the column name from "Style" to "Property" and "Default Value" to "Configuration Options"
   -  Update the content in the "Description" and "Configuration Options" for the **Visibility** property. [reference](https://docs.tooljet.com/docs/widgets/rich-text-editor#styles) ( Please just check the sentence and table structure from the given reference )
   -  Add the Box shadow property to the table. [reference](https://docs.tooljet.com/docs/widgets/rich-text-editor#styles) ( Please add the property to the table similar to that given in the reference )

**The changes are to be implemented in two files -** 
 Next - ToolJet/docs/docs/widgets/svg-image.md
 Version 2.50.0 (LTS) - ToolJet/docs/versioned_docs/version-2.50.0-LTS/widgets/svg-image.md

### If the issue is confirmed, would you be willing to submit a pull request?

No

### Code of Conduct

- [X] I agree to follow the ToolJet Code of Conduct

### PR DESCRIPTION
**Pull request by vroomvee**
I have updated the content according to the requirements mentioned in the issue. I am a beginner and this is my first open-source contribution ever, I hope you excuse my mistakes made inadvertently, if any. Looking forward to some feedback and tips as well. Thank you for this opportunity!

----
will close #11116

### CODE REVIEW

**Comment by PriteshKiri** [comment]
Solves #11116

**Comment by RudraPratik30** [comment]
Hey @vroomvee, thanks for contributing!
As per the issue, you were supposed to make changes in two versions (Next and LTS). But I can see you have only edited the LTS version, can you please edit the Next version as well? 
Please refer the [issue](https://github.com/ToolJet/ToolJet/issues/11116) for file path.
You can push the changes in same PR.

**Comment by PriteshKiri** [comment]
Hey @vroomvee 

Can you please check the comment by @RudraPratik30 and make the changes in the same PR?

**Comment by vroomvee** [comment]
> Hey @vroomvee
> 
> Can you please check the comment by @RudraPratik30 and make the changes in the same PR?

Yes, on it right now!

File: docs/docs/widgets/svg-image.md [hunk 1]
```diff
@@ -56,21 +56,23 @@ Under the <b>General</b> accordion, you can set the value in the string format.
...
 
-## Layout
+## Devices
 
-| <div style={{ width:"100px"}}> Layout </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
+| <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
 |:--------------- |:----------------------------------------- | :------------------------------------------------------------------------------------------------------------- |
-| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on **fx** to set the value `{{true}}` or `{{false}}`. |
-| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on **fx** to set the value `{{true}}` or `{{false}}`. |
+| Show on desktop | Makes the component visible in desktop view. | You can set it with the toggle button or dynamically configure the value by clicking on fx and entering a logical expression. |
```
Comment (RudraPratik30) [inline docs/docs/widgets/svg-image.md:63]: Hey @vroomve fx should be in bold, can you please update this in both rows and both versions?

File: docs/docs/widgets/svg-image.md [hunk 2]
```diff
@@ -56,21 +56,23 @@ Under the <b>General</b> accordion, you can set the value in the string format.
...
+---
+
 ## Styles
 
-| <div style={{ width:"100px"}}> Style </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"100px"}}> Default Value </div> |
+| <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"100px"}}> Configuration Options </div> |
 |:------------ |:-------------|:--------- |
-| Visibility | Toggle on or off to control the visibility of the component. You can programmatically change its value by clicking on the **fx** button next to it. If `{{false}}` the component will not be visible after the app is deployed. | By default, it's set to `{{true}}`. |
-
+| Visibility | Controls component visibility. Toggle or set dynamically. | Enable/disable the toggle button or dynamically configure the value by clicking on fx and entering a logical expression. |
```
Comment (RudraPratik30) [inline docs/docs/widgets/svg-image.md:76]: Here as well, the fx needs to be in bold

**Comment by vroomvee** [comment]
@RudraPratik30 is everything good now? i pushed changes in the Next version as well.

File: svg-image.md
Comment (RudraPratik30) [inline svg-image.md:1]: I think you created this file by mistake, can you please delete it?

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @vroomvee can you please just delete that file? Rest of the changes look good to me!

File: svg-image.md
Comment (vroomvee) [reply svg-image.md:1]: oh gosh yes okay, im sorry im still figuring all this out!

**Review by RudraPratik30 — APPROVED** [review]
Hey @vroomvee, thanks for contributing!
The changes look good to me and forwarding it to the team for final review.

**Review by karan-rathod-316 — APPROVED** [review]
Looks good to me!

**Review by adishM98 — APPROVED** [review]

**Pull request merged**